meta/runtime.yml

```diff
@@ -271,17 +271,3 @@ plugin_routing:
     execute_lambda:
       # Deprecation for this alias should not *start* prior to 2024-09-01
       redirect: community.aws.lambda_execute
-    iam_cert:
-      redirect: community.aws.iam_server_certificate
-      deprecation:
-        removal_version: 5.0.0
-        warning_text: >-
-          iam_cert has been renamed to iam_server_certificate for consistency.
-          Please update your tasks.
-    rds_snapshot:
```

Comment:
Don't we have to also remove those from the action group?

Reply:
Missed this comment.  Yes you're right, I missed that.